debian/control

```diff
@@ -4,18 +4,23 @@ Priority: extra
 Maintainer: Sipwise Development Team <support@sipwise.com>
 Build-Depends: debhelper (>= 5),
                iptables-dev (>= 1.4),
-               libavcodec-dev,
-               libavfilter-dev,
-               libavformat-dev,
-               libavresample-dev,
-               libavutil-dev,
+               libavcodec-dev (>= 6:10),
+               libavcodec55 (>= 6:10) | libavcodec-dev (>= 6:11),
+               libavfilter-dev (>= 6:10),
+               libavfilter4 (>= 6:10) | libavfilter-dev (>=6:11),
+               libavformat-dev (>= 6:10),
+               libavformat55 (>= 6:10) | libavformat-dev (>=6:11),
+               libavresample-dev (>= 6:10),
+               libavutil-dev (>= 6:10),
                libcurl4-openssl-dev | libcurl4-gnutls-dev | libcurl3-openssl-dev | libcurl3-gnutls-dev,
                libevent-dev (>= 2.0),
                libglib2.0-dev (>= 2.30),
                libhiredis-dev,
+               libopus0 (>= 1.1),
                libpcap0.8-dev | libpcap-dev,
                libpcre3-dev,
                libssl-dev (>= 1.0.1),
+               libswscale2 (>= 6:10) | libswscale-dev (>=6:11),
```

Comment:
I don't see the code making direct use of any of the stuff provided by libswscale. This dependency seems gratuitous here?